spec/lcm/integration/lcm_end_to_end_spec.rb

```diff
@@ -6,27 +6,42 @@
 require_relative 'support/constants'
 require_relative 'support/configuration_helper'
 require_relative 'support/lcm_helper'
+require_relative 'brick_runner'
 require_relative 'shared_examples_for_synchronization_bricks'
-require_relative 'shared_contexts_for_lcm_specs'
+require_relative 'shared_contexts_for_lcm'
 
 # global variables to simplify passing stuff between shared contexts and examples
 $master_projects = []
 $client_projects = []
 $master = false
 
+release_additional_hidden_params = {
+    hidden_msg_from_release_brick: 'Hi, I was set by release brick but keep it secret',
+    SECURE_PARAM_2: 'I AM SET TOO'
+}
+
+rollout_additional_hidden_params =  {
```

Comment:
warning: Style/SpaceAroundOperators: Operator `=` should be surrounded by a single space. (rubocop)